app/views/documents/edit.html.erb

```diff
@@ -33,14 +32,32 @@
 
 <script src="https://js.shubox.io/v1/<%= Rails.application.secrets.shubox_js_id %>.js"></script>
 
+<script id="template" type="text/x-handlebars-template">
```

Comment:
would it make sense to use an id or class that somehow indicates we are using this for js purposes? Not sure how you feel about js prefixed. Alternatively, maybe a name more specific than `template` would be helpful.